.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,59 @@
+---
+name: 🐛 Bug report
+about: Create a report to help us improve
+title: '🐛 '
+labels: 'bug'
+assignees: ''
+
+---
+<!---
+Please DO NOT remove any fields from this template. If there is nothing to add, leave N/A.
+--->
+### Operator Version, Kind and Kubernetes Version
+ - Operator version:
+ - Kind: Workspace | Module
+ - Kubernetes version:
+
+### YAML Manifest File
+```yaml
+# Copy-paste your YAML manifest here
+```
+
+### Output Log
+<!---
+Please provider a link to a GitHub Gist containing the complete output log.
+Please DO NOT paste the debug output in the issue; just paste a link to the Gist.
+--->
+
+### Kubectl Outputs
```

Comment:
What is this asking for exactly? Which kind of outputs? Console output of kubectl commands? The use of plural (outputs) is confusing me a bit.

Reply:
This is for `kubectl` outputs that are related to the reported issue. For instance: `kubectl get module`, `kubectl get workspace`, `kubectl get secret`, `kubectl get configmap`, and the same but `kubectl describe`, instead of `kubectl get`.
An artificial example here can be incorrect storage for the outputs, like "It says 'sensitive', but it appears in the configmap". 🤨 In this case, great to see `get workspace | configmap | secret`.

Reply:
Got it. May I suggest to rephrase this to "Output of relevant `kubectl` commands" ?

Reply:
Yes sir! Done. :) That makes more sense, thanks!